ansible/playbooks/jenkins/worker/fix.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+#
+# Fix common problems with Jenkins workers
+#
+
+- hosts:
+  - test
+  - release
+  # Benchmarking machine
+  - infra-softlayer-ubuntu1404-x64-2
+
+  tasks:
+    # Clean disk space
+    - name: Clean disk space
+      block:
+        - name: Get disk space
+          command: "df -h"
+          register: disk_usage
+        - set_fact:
+            disk_usage: "{{ disk_usage.stdout_lines }}"
+        - debug:
+            var: "{{ disk_usage }}"
+
+        - name: Get directories to delete
+          find:
+            paths: /home/iojs/build/workspace
+            patterns: "node-test-binary-arm,node-test-commit-*,libuv-test-commit-*,citgm-*"
+            recurse: no
+            file_type: directory
+          register: directories
+
+        - name: Remove directories
+          file:
+            path: "{{ item.path }}"
+            state: absent
+          with_items: "{{ directories.files }}"
```

Comment:
The best way to delete would be to move to a temporary directory in the same filesystem (inside the workspace should be safe), and then delete that directory. I've seen issues in the Pis where a file can't be deleted because of filesystem errors. Moving first at least ensures the original directories can be created again clean. I understand this requires some more work, so I'm just mentioning this here and completely ok to leave this to another PR sometime. Please don't block this PR on this.

Reply:
> I've seen issues in the Pis where a file can't be deleted because of filesystem errors.

That usually happens if there are hanging `node` processes (live or undead).
I support the move->delete strategy anyway.